### ISSUE TITLE
Fix missing AdminSecurity annotations for 8.1.0

### ISSUE THREAD

**Issue by matks** [PrestaShop/PrestaShop#32575]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Release task

This is one of the last steps to do for 8.1.0 Release Candidate.

This ticket is about performing this step from the https://www.prestashop-project.org/maintainers-guide/processes/release/prestashop/preliminary-tasks/ process.

The Command `php bin/console prestashop:linter:security-annotation find-missing` helps us finding controllers that do NOT have the Annotation `@AdminSecurity()` needed to secure them. This is a [system](https://devdocs.prestashop-project.org/8/development/architecture/modern/controller-routing/#how-does-this-work) which uses back-office user privileges to check if user can access a given controller action.

For some controllers that is expected: it is because they are either fully open or are protected using something else than `@AdminSecurity()`.

## What to do

It is necessary, to make sure 8.1.0 Release Candidate is secure, to go through the list below (this is the output of the Command, when run on `8.1.x` branch) and verify for each line:
- is it expected that this Controller has no `@AdminSecurity`; is it because it is an expected exception to the rule
- if not, please add the missing `@Security()` annotation (see example PR https://github.com/PrestaShop/PrestaShop/pull/21190/)

**Comment by matks** [comment]
Output of the Command

```
 * admin_common_notifications
 * admin_common_notifications_ack
 * admin_common_pagination
 * admin_common_sidebar
 * admin_common_reset_search
 * admin_common_reset_search_by_filter_id
 * admin_security_compromised
 * admin_product_new
 * admin_product_form
 * admin_product_virtual_save_action
 * admin_product_virtual_remove_file_action
 * admin_product_virtual_download_file_action
 * admin_product_virtual_remove_action
 * admin_product_image_upload
 * admin_product_image_positions
 * admin_product_image_form
 * admin_product_image_delete
 * admin_category_simple_add_form
 * admin_get_ajax_categories
 * admin_feature_get_feature_values
 * admin_supplier_refresh_product_supplier_combination_form
 * admin_warehouse_refresh_product_warehouse_combination_form
 * admin_product_catalog_filters
 * admin_product_list
 * admin_product_bulk_action
 * admin_product_unit_action
 * admin_product_mass_edit_action
 * admin_product_export_action
 * admin_import_data_configuration_index_redirect
 * admin_import_file_upload
 * admin_emails_send_test
 * admin_shops_search
 * admin_authorized_applications_index
 * admin_authorized_applications_view
 * admin_authorized_applications_create
 * admin_authorized_applications_edit
 * admin_authorized_applications_delete
 * admin_api_accesses_create
 * admin_employees_edit
 * admin_employees_toggle_navigation
 * admin_employees_change_form_language
 * admin_international_translation_overview
 * admin_currencies_update_live_exchange_rates
 * admin_country_states
 * admin_countries_edit
 * admin_module_manage_action
 * admin_module_manage_action_bulk
 * admin_module_manage_update_all
 * admin_module_import
 * admin_module_notification_count
 * admin_theme_customize_layouts
 * admin_mail_theme_save_configuration_deprecated
 * admin_mail_theme_send_test_mail_deprecated
 * admin_mail_theme_send_test_module_mail_deprecated
 * admin_carriers_edit
 * admin_prestashopexamplemodule_module_upgrade
```

**Comment by matks** [comment]
Fixed by @boherm in https://github.com/PrestaShop/PrestaShop/pull/32609

**Comment by boherm** [comment]
> Fixed by @boherm in #32609

And above all by @matks 😄
